tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -287,8 +287,13 @@ def __init__(self, suite_name, test_queue, test_source_cls, browser_cls, browser
 
         self.test_source = test_source_cls(test_queue)
 
+        self.manager_number = next_manager_number()
         self.browser_cls = browser_cls
-        self.browser_kwargs = browser_kwargs
+        self.browser_kwargs = browser_kwargs.copy()
+        if self.browser_kwargs.get("device_serial"):
+            # Assign Android device to runner according to manager_number
+            self.browser_kwargs["device_serial"] = (
+                self.browser_kwargs["device_srial"][self.manager_number-1])
```

Comment:
There's a typo here. Given that, is the code really needed, if everything seemed fine with "device_srial"?

Reply:
Thanks foolip@. Yes this is a typo. When I tested this locally I did met issue and fixed it, but at a different repo. Looks I forget to propagate the change back.